### PR TITLE
EVENT-643: Include Group Link for All Group Registrants on Registrations Page Group Column

### PR DESCRIPTION
**Pull request by katefree**

### CODE REVIEW

File: app/scripts/controllers/eventRegistrations.js [hunk 1]
```diff
@@ -474,6 +474,15 @@ angular
       return `${registrant.firstName} ${registrant.lastName}`;
     };
 
+    $scope.isGroupRegistrant = function(registrant) {
```
Comment (TheNoodleMoose) [inline app/scripts/controllers/eventRegistrations.js]: Can you make this an arrow function?

File: test/spec/controllers/eventRegistrations.spec.js [hunk 2]
```diff
@@ -104,4 +104,28 @@ describe('Controller: eventRegistrations', function() {
     scope.registerUser();
     expect(openModal).toHaveBeenCalled();
   });
+
+  it('isGroupRegistrant should return true for primary user group registrations', function() {
```
Comment (TheNoodleMoose) [inline test/spec/controllers/eventRegistrations.spec.js]: Can you change this to an arrow function as well?

File: test/spec/controllers/eventRegistrations.spec.js [hunk 3]
```diff
@@ -104,4 +104,28 @@ describe('Controller: eventRegistrations', function() {
...
+    let registrant = {
+      registrantTypeId: '47de2c40-19dc-45b3-9663-5c005bd6464b',
+    };
+    let result = scope.isGroupRegistrant(registrant);
+    expect(result).toBeTruthy();
+  });
+
+  it('isGroupRegistrant should return true for dependent user group registrations', function() {
```
Comment (TheNoodleMoose) [inline test/spec/controllers/eventRegistrations.spec.js]: here

File: test/spec/controllers/eventRegistrations.spec.js [hunk 4]
```diff
@@ -104,4 +104,28 @@ describe('Controller: eventRegistrations', function() {
...
+    let registrant = {
+      registrantTypeId: '67c70823-35bd-9262-416f-150e35a03514',
+    };
+    let result = scope.isGroupRegistrant(registrant);
+    expect(result).toBeTruthy();
+  });
+
+  it('isGroupRegistrant should return false for not dependent and not primary user group registrations', function() {
```
Comment (TheNoodleMoose) [inline test/spec/controllers/eventRegistrations.spec.js]: and here

File: app/scripts/controllers/eventRegistrations.js (hunk 1 above)
Comment (katefree) [reply app/scripts/controllers/eventRegistrations.js]: fixed

File: test/spec/controllers/eventRegistrations.spec.js (hunk 2 above)
Comment (katefree) [reply test/spec/controllers/eventRegistrations.spec.js]: fixed

File: test/spec/controllers/eventRegistrations.spec.js (hunk 3 above)
Comment (katefree) [reply test/spec/controllers/eventRegistrations.spec.js]: fixed

File: test/spec/controllers/eventRegistrations.spec.js (hunk 4 above)
Comment (katefree) [reply test/spec/controllers/eventRegistrations.spec.js]: fixed

**Comment by hlbraddock** [comment]
@TheNoodleMoose Approved?

**Pull request merged**